test/v4/retrieveAnnualSubmission/def3/model/response/Def3_RetrieveAnnualSubmissionResponseSpec.scala

```diff
@@ -29,8 +28,6 @@ class Def3_RetrieveAnnualSubmissionResponseSpec
     with Def3_RetrieveAnnualSubmissionFixture
     with MockSeBusinessFeatureSwitches {
```

Comment:
Please remove `with MockSharedAppConfig with MockSeBusinessFeatureSwitches` as it is not required, thanks.